File: app/src/main/java/com/example/plan_de_ahorro/data/model/Payment.kt
package com.example.plan_de_ahorro.data.model

import com.google.gson.annotations.SerializedName

data class Payment(
    @SerializedName("_id") val id: String? = null,
    val amount: Double,
    val memberId: String,
    val planId: String,
    val date: String? = null
)

File: backend_parcial/models/Payment.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const PaymentSchema = new Schema({
  memberId: { type: Schema.Types.ObjectId, ref: 'Member', required: true },
  planId: { type: Schema.Types.ObjectId, ref: 'Plan', required: true },
  amount: { type: Number, required: true },
  date: { type: Date, default: Date.now }
});

module.exports = mongoose.model('Payment', PaymentSchema);
